demo/admin/src/products/future/generated/ProductsGrid.tsx

```diff
@@ -159,17 +154,11 @@ export function ProductsGrid({ filter }: Props): React.ReactElement {
                         </IconButton>
                         <CrudContextMenu
                             copyData={() => {
-                                const row = params.row;
+                                // Don't copy id, because we want to create a new entity with this data
+                                const { id, ...filteredData } = objectFilter(productsFragment, params.row);
```

Comment:
I'm still unsure if I like this 😅

Reply:
I didn't put a lot of effort into this because there is already a known issue. This needs to be fixed with an independent query, requiring to rewrite a lot more. I wouldn't have changed it if I there weren't typescript errors.